### PR TITLE
💄 Study File List sort status of changes needed to the top

### LINKED ISSUE #383: Study File List sort status of changes needed to the top
It would be convenient to have files with the status of `Changes Needed` sorted to the top of the list as when there are multiple pages it could be frustrating to have to search for the ones marked that needed changes 

--- 

> the file that needs changes is on the second page 
Screen Shot 2019-07-22 at 4 07 58 PM

### PR DESCRIPTION
**Pull request by XuTheBunny**
Before sorting:
image

After sorting:
image

Emphasizing "Changes Needed" status:
image

Closes #383

### CODE REVIEW

File: src/components/FileList/FileList.js [hunk 1]
```diff
@@ -20,38 +20,35 @@ const FileList = ({fileList, studyId}) => {
   const handlePageClick = (e, {activePage}) => {
     setPage(activePage);
   };
-  const pageCount = Math.ceil(fileList.length / perPage);
-  const pageItems = fileList.slice(
+  var fileListCopy = JSON.parse(JSON.stringify(fileList));
+  var statusList = [];
+  var i;
+  for (i in fileListCopy) {
```
Comment (dankolbman) [inline src/components/FileList/FileList.js]: I think you may be able to do this with the `sort` function:
```
fileListCopy.sort((el1, el2) => {
    let el1Status = fileSortedVersions[0].node.state;
    return el1Status === "CHN" ? 1 : 0
})
```

File: src/views/NewDocumentView.js [hunk 2]
```diff
@@ -59,7 +59,12 @@ const NewDocumentView = ({match, history, location, createDocument}) => {
       </Container>
       <Segment.Group>
         <Segment>
-          <Header as="h4">Uploaded File: {location.state.file.name}</Header>
+          <Header as="h4" title={location.state.file.name.length}>
+            Uploaded File:{' '}
+            {location.state.file.name.length > 80
```
Comment (bdolly) [inline src/views/NewDocumentView.js]: could possibly make this a truncate utility function but not a big deal

File: src/components/FileList/FileList.js (hunk 1 above)
Comment (bdolly) [reply src/components/FileList/FileList.js]: 👍

File: src/components/FileList/FileElement.js [hunk 3]
```diff
@@ -89,24 +92,17 @@ const FileElement = ({fileNode, loading, history, match, fileListId}) => {
...
-        <p>
-          {fileDescription ? (
-            <>
-              {fileDescription.length > 100
-                ? fileDescription.substring(0, 100) + '...'
-                : fileDescription}
-            </>
-          ) : null}
+        <p className="noMargin">
+          {fileDescription ? <>{lengthLimit(fileDescription, 100)}</> : null}
```
Comment (bdolly) [inline src/components/FileList/FileElement.js]: niiice 👍

**Pull request merged**